app/config/views.yml

```diff
@@ -10,6 +10,9 @@ ezpublish:
                     home:
                         controller: ez_query:LocationQueryAction
                         params:
+                            omitted_location_ids:
```

Comment:
Better than ids, we can define content type identifiers.
This should be in a different PR